packages/python-galaxy-ng/python-galaxy-ng.spec

```diff
@@ -1,14 +1,16 @@
 # Created by pyp2rpm-3.3.3
 %global pypi_name galaxy-ng
 
+%global full_version %{version}a1
```

Comment:
Is it time for a `pre_release` macro as we have it in Foreman? I can imagine this would show up more often.

Reply:
probably, yeah. I have a "do nightlies for some core packages", so I'd add that then.

Reply:
Fair enough